site/src/pages/RoadmapPage/transfers/APExamsSection.tsx

```diff
@@ -1,17 +1,42 @@
-import { FC, useState } from 'react';
+import { FC, useState, useContext, useEffect, useCallback } from 'react';
 import MenuSection, { SectionDescription } from './MenuSection';
 import MenuTile from './MenuTile';
+import Select from 'react-select';
+import ThemeContext from '../../../style/theme-context';
+import { comboboxTheme } from '../../../helpers/courseRequirements';
+import trpc from '../../../trpc';
+import { useAppDispatch } from '../../../store/hooks';
 
 interface ScoreSelectionProps {
   score: number | null;
   setScore: (value: number) => void;
 }
 
+interface APExam {
+  fullName: string;
+  catalogueName: string;
+  rewards: {
+    acceptableScores: number[];
+    unitsGranted: number;
+    electiveUnitsGranted: number;
+    geCategories: string[];
+    coursesGranted: {
+      string: string[];
+    };
+  }[];
+}
+
 const ScoreSelection: FC<ScoreSelectionProps> = ({ score, setScore }) => {
   /** @todo style this according to the Figma, add options & optgroup titled 'Score' */
   return (
     <select value={score ?? ''} onInput={(event) => setScore(parseInt((event.target as HTMLInputElement).value))}>
-      <option>1</option>
+      <optgroup label="Score">
+        <option value="1">1</option>
+        <option value="2">2</option>
+        <option value="3">3</option>
+        <option value="4">4</option>
+        <option value="5">5</option>
```

Comment:
If I'm not mistaken, `<option>` tags without values default to their content, so you could do `<option>1</option>`